pybamm/models/full_battery_models/lead_acid/base_lead_acid_model.py

```diff
@@ -34,8 +34,22 @@ def default_geometry(self):
 
     @property
     def default_var_pts(self):
+        # Choose points that give uniform grid for the standard parameter values
```

Comment:
makes sense. is this the case for li-ion too?

Reply:
Thought I needed this to make one of the tests work but it turned out to be an issue with the solver. I guess we could add functionality so that the user can pass in grid points for, say, x_n, and then "uniform" for the other two, and it automatically selects the right number of points to make the grid uniform?